server/sonar-server/src/test/java/org/sonar/server/user/UserUpdaterTest.java

```diff
@@ -125,6 +141,14 @@ public void create_user() throws Exception {
     assertThat(dto.getCreatedAt()).isEqualTo(1418215735482L);
     assertThat(dto.getUpdatedAt()).isEqualTo(1418215735482L);
     assertThat(result).isFalse();
+
+    List<SearchHit> indexUsers = es.getDocuments(UserIndexDefinition.INDEX, UserIndexDefinition.TYPE_USER);
```

Comment:
FYI, you could also have used "public <E extends BaseDoc> List<E> getDocuments(String indexName, String typeName, final Class<E> docClass)", it would allow you to return UserDoc.
No need to do anything.